config/src/main/java/com/quorum/tessera/config/ServerConfig.java

```diff
@@ -48,20 +48,25 @@
     @XmlElement
     private String serverAddress;
 
+    @XmlElement
+    private String corsDomains;
+
     public ServerConfig(final AppType app,
                         final boolean enabled,
                         final String serverAddress,
                         final CommunicationType communicationType,
                         final SslConfig sslConfig,
                         final InfluxConfig influxConfig,
-                        final String bindingAddress) {
+                        final String bindingAddress,
+                        final String corsDomains) {
```

Comment:
Domains is a list of things not just a string

Reply:
yes, it is interpreted as a comma separated list of domains - see initCorsDomains.

Reply:
exactly.